less/type.less

```diff
@@ -238,6 +238,67 @@ ol {
   }
 }
 
+ul.tw-list-styled {
+  padding: 0;
+  margin: 32px 0;
+  list-style-type: none;
+  li {
+    position: relative;
+    padding-left: 40px;
+    margin-bottom: 16px;
+    counter-increment: step-counter;
+    strong {
+      color: #2ED06E;
+    }
+    &::before {
+      font-family: "TransferWise-IconFont";
+      position: absolute;
+      left: 0;
+      width: 24px;
+      content: "\e916";
+      line-height: 22px;
+      margin-right: 16px;
+      font-size: 16px;
+      text-align: center;
+      color: #2ED06E;
+      font-weight: bold;
+      padding: 0px 3px;
+      border: 1px solid #E2E6E8;
```

Comment:
Can we use the LESS variables for colours? e.g. `@brand-smoke-plus-10` it means we can keep them all in line if (or more like when...) design chooses to tweak the palette.